### ISSUE TITLE
Gauge is not appearing on participez.nanterre participatory budget

### ISSUE THREAD

**Issue by virgile-dev** [OpenSourcePolitics/decidim#717]
**Probleme description**
It appears that participez.nanterre.fr is not up to date with the last version. (That's @valentinchaput diagnostic).
Symptom is the gauge is not showing nor on the project list or on the project pages.
<img width="1113" alt="Capture d’écran 2019-06-18 à 15 44 53" src="https://user-images.githubusercontent.com/11473995/59689780-8ca77f80-91e0-11e9-9353-b2e5a3f26167.png">
<img width="1114" alt="Capture d’écran 2019-06-18 à 15 44 46" src="https://user-images.githubusercontent.com/11473995/59689781-8ca77f80-91e0-11e9-96d9-01fdfeaf69f2.png">

**Proper behaviour can be seen on Roubaix**
https://budgetparticipatif.ville-roubaix.fr/processes/pcp2019/f/10/projects/25
<img width="1111" alt="Capture d’écran 2019-06-18 à 15 47 39" src="https://user-images.githubusercontent.com/11473995/59689766-85807180-91e0-11e9-81ea-89c00bc8a41b.png">
<img width="1106" alt="Capture d’écran 2019-06-18 à 15 47 31" src="https://user-images.githubusercontent.com/11473995/59689768-86b19e80-91e0-11e9-8932-a9d8ab0f6cdb.png">

**Comment by moustachu** [comment]
@virgile-dev It's not clear what to be fix in this case :
- vote per project
- 1 project max

Is it pertinent to display a progress bar for 1 project ? 

Please provide the expected behavior and translations ;)

**Comment by virgile-dev** [comment]
@valentinchaput can you please chime in ? I haven't contributed to the specs and implementation.

**Comment by valentinchaput** [comment]
Two separated issues with a same answer:
1. There is a bug on Nanterre's platform because the progress bar is not diplayed. I guess it means we did not deploy the recent enhancements made by Armand (email confirmation & green layer after voting...). 
2. As Nanterre is using a vote with a single project to choose, the optimized wording we deployed on Roubaix the other day should be clearer. 
>> Can we have both the feature & translations updated @moustachu?
